editor.planx.uk/src/lib/graphql.ts

```diff
@@ -34,11 +37,62 @@ const customFetch = async (
   return fetchResult;
 };
 
-const authHttpLink = createHttpLink({
+const httpLink = createHttpLink({
   uri: process.env.REACT_APP_HASURA_URL,
   fetch: customFetch,
 });
 
+/**
+ * Set auth header in Apollo client
+ * Must be done post-authentication once we have a value for JWT
+ */
+export const authMiddleware = setContext(async () => {
+  const jwt = await getJWT();
+
+  return {
+    headers: {
+      authorization: jwt ? `Bearer ${jwt}` : undefined,
+    },
+  };
+});
+
+const authHttpLink = authMiddleware.concat(httpLink);
+
+/**
+ * Authenticated web socket connection - used for GraphQL subscriptions
+ */
+const authWsLink = new WebSocketLink({
+  uri: `ws://localhost:7100/v1/graphql`,
```

Comment:
might be convenient/consistent to add this to editor .env (not secret!) as: 
```
REACT_APP_HASURA_WEBSOCKET=ws://localhost:7100/v1/graphql
```

Reply:
Good spot - had intended to fix this! I'll add a variable for this now.

Reply:
Fixed in [456eaed](https://github.com/theopensystemslab/planx-new/pull/3002/commits/456eaed695d00d6d422115074e4c6ac4dc663097)

Reply:
Just checked this out - we don't actually need to update `.env.example` here as this is an env var within the React app only (and it's a genuine env var, not a secret).